src/main/java/ecdar/controllers/EcdarController.java

```diff
@@ -889,6 +905,8 @@ private void createNewProject() {
 
         Ecdar.projectDirectory.set(null);
 
+        Ecdar.isSplitProperty().set(false);
+
         projectPane.getController().resetProject();
```

Comment:
If "setProjectDirectory(Path path);" is created then it could also handle this reset.

Reply:
:heart:

Reply:
:smiling_face_with_three_hearts: